Apps.AIUtilities/Actions/PromptActions.cs

```
@@ -134,17 +124,17 @@ public async Task<PromptResponse> Translate([ActionParameter] TranslateRequest i
     public PromptResponse GetLocalizableContentFromImage()
         => new(Prompts.GetLocalizableContentFromImage);
 
-    private async Task<string?> BuildPromptFromInputs(string? text, FileReference? textFile)
+    private async Task<string?> BuildPromptFromInputs(string? text, FileReference? textFile) 
     {
-        if (text is null && textFile is null)
-            return null;
+        if (string.IsNullOrEmpty(text) || textFile is null)
```

Why did you choose to use the `||` (OR) condition here? From my understanding, the requirement is to validate that **both** `text` and `textFile` are empty. If that is the case, we should throw an exception, since the user must provide at least one non-empty input (either a file or text).